pkg/ccl/kvccl/kvfollowerreadsccl/followerreads_test.go

```diff
@@ -847,15 +847,24 @@ func TestFollowerReadsWithStaleDescriptor(t *testing.T) {
 	// load based rebalancing to make sure it doesn't move.
 	kvserverbase.LoadBasedRebalancingMode.Override(ctx, &settings.SV, kvserverbase.LBRebalancingOff)
 
+	// NB: Tenants need capabilities to be able to relocate ranges.
+	if !tc.Server(0).DeploymentMode().IsSingleTenant() {
+		require.NoError(t, tc.Server(0).GrantTenantCapabilities(
+			context.Background(), serverutils.TestTenantID(),
+			map[tenantcapabilitiespb.ID]string{
+				tenantcapabilitiespb.CanAdminRelocateRange: "true",
+			}))
+	}
```

Comment:
Not for this PR: But I wonder how many more tests would "just work" if we granted everything at cluster startup.